windows bat/PokemonGo-Bot-Install.bat

```diff
@@ -0,0 +1,125 @@
+@echo off

```

Comment:
Still seems like bad form to have this in the repo. It would be harder for non Windows users to update it and edit. It should be pretty reasonable to just use an underscore, right?